package.json

```
@@ -5,7 +5,7 @@
 	"author": "HPI Schul-Cloud",
 	"private": true,
 	"scripts": {
-		"dev:nuxt": "cross-env NODE_ENV=development nuxt --port 4000",
+		"dev:nuxt": "API_URL=http://localhost:3030 FEATURE_TEAMS_ENABLED=true nodemon --inspect --watch src/ --ignore src/legacy-client src/server",
```

```suggestion
		"dev:nuxt": "nodemon --inspect --watch src/ --ignore src/legacy-client src/server",
```
1. `API_URL` has a default value which is the same as you specified here.
2. if we need to set such variables for all developers, we should move it to an env file (https://stackoverflow.com/questions/50828904/using-environment-variables-with-vue-js)

---

Alright I removed it!